src/components/leave-room-button.js

```diff
@@ -0,0 +1,16 @@
+AFRAME.registerComponent("leave-room-button", {
+  init() {
+    this.onClick = () => {
+      console.log("click");
+      this.el.sceneEl.emit("leave_room_requested");
```

Comment:
nit: We seem to alternate between using hyphens and underscores in our event names. Would be nice to settle on one of them. Looks like underscores are winning, so just something to keep an eye out for in other reviews.

Reply:
ah yeah i've noticed this too